src/main/java/org/jabref/logic/shared/DBMSProcessor.java

```diff
@@ -305,15 +299,15 @@ protected void insertIntoFieldTable(List<BibEntry> bibEntries) {
      * Updates the whole {@link BibEntry} on shared database.
      *
      * @param localBibEntry {@link BibEntry} affected by changes
-     * @throws SQLException
+     * @throws SQLException in case of error
```

Comment:
This comment seems to be a bit... ... ... dark 😜

Reply:
intelij was complaiing that the javadoc has no description^^